custom_components/homewhiz/__init__.py

```diff
@@ -88,11 +89,31 @@ async def connect(self):
 
         return True
 
+    async def try_reconnect(self):
+        while self._conn is None or not self._conn.is_connected:
+            if not bluetooth.async_address_present(
+                self.hass, self.address, connectable=True
+            ):
+                _LOGGER.info(
+                    f"[{self.address}] Device not found. "
+                    f"Will reconnect automatically when the device becomes available"
+                )
+                return
+            try:
+                await self.connect()
+            except Exception:
+                _LOGGER.info(
+                    f"[{self.address}] Can't reconnect. Waiting a minute to try again"
+                )
+                time.sleep(60)
```

Comment:
Shouldn't this be an asyncio.sleep rather than a time sleep?

Reply:
You are right. I didn't know about this, thanks!